tests/e2e/specs/loans-spec.js

```diff
@@ -0,0 +1,71 @@
+import LoansPage from '../pages/loans-page';
+
+const loansPage = new LoansPage();
+
+describe('Loans', () => {
+	before(() => {});
+	context('Page', () => {
+		it('should be accessible from sidenav', () => {
+			cy.visit('/');
+			cy.findByTestId('sidenav-/loans').click();
+			cy.findByTestId('loans-form', { timeout: 30 * 1e3 }).should('be.visible');
+		});
+		it.only('should create & manage loans', () => {
```

Comment:
is `it.only` on purpose here?

Reply:
it should be separated to two tests; you can use `context()` for example (or use different`*-spec.js` files and do the preparation for manage loan test inside `before()`):
1. should create loan
2. should manage loan? (needs more descriptive name)
```
describe('Loans tests', () => {
	before(() => {
	...
	});
	context('Create a loan', () => {
		it(`should ..`, () => {
		});
		it(`should ..`, () => {
		});
		...
	});
	context('Manage a loan', () => {
		it(`should ..`, () => {
		});
		it(`should ..`, () => {
		});
		...
	});
});
```

Reply:
This one as well. Best to run it in one test case to speed up testing.

Reply:
Did split the test into smaller cases.